### ISSUE TITLE
Open database via intent

### ISSUE THREAD

**Issue by Exeteres** [Kunzisoft/KeePassDX#178]
Can i send intent from my app with database path and password and get KeePassDX window with unlocked database?

**Comment by J-Jamet** [comment]
No, it is not a feature implemented but it is a good idea, we should make an opening API for other applications with good documentation. But first I need to develop more urgent features (sessions, attachments ...).

**Comment by lollita** [comment]
Up!
